torch/csrc/utils/python_mode.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <ATen/core/PythonModeTLS.h>
+
+namespace torch {
+namespace python_dispatch {
+
+struct StashPythonModeGuard {
```

Comment:
I guess this naming isn't symmetric with the torch function guard. Do you prefer Stash over No?

Reply:
I matched the name to [this guard in PythonFallback](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/PythonFallbackKernel.cpp#L30-L44) that has the same pattern. I guess I prefer it slightly but don't feel strongly. I can change torch function to be stash as well